crm_rma_stock_location/__openerp__.py

```diff
@@ -1,49 +1,22 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Author: Guewen Baconnier
-#    Copyright 2014 Camptocamp SA
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# © 2014-2016 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
 
 {'name': 'RMA Stock Location',
- 'version': '1.0',
+ 'version': '9.0.1.0.0',
  'author': "Camptocamp,Odoo Community Association (OCA)",
```

Comment:
Could you check the last author from 8.0?. Please

https://github.com/OCA/rma/blob/dd9fcd35744fb683330dcc71cb86e66fb09116e0/crm_rma_stock_location/__openerp__.py#L30